samples/app-anonymous-users/nodejs/README.md

```diff
@@ -0,0 +1,295 @@
+---
+page_type: sample
+description: This sample shows anonymous users support in Teams meeting apps.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "22-02-2023 10:00:01"
+urlFragment: officedev-microsoft-teams-samples-app-anonymous-users-nodejs
+
+---
+
+## Anonymous User Support
+
+This sample shows anonymous users support in Teams meeting apps.
+
+**Interaction with app**
+![appanonymoususersGif](Images/anonymoususersupport.gif)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org)
+
+    ```bash
+    # determine node version
+    node --version
+    ```
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## Setup
+
+> Note these instructions are for running the sample on your local machine.
+
+1. Run ngrok - point to port 3978
+
+   ```bash
+     ngrok http -host-header=rewrite 3978
```

Comment:
host-header option needs to be preceded by double dash "--"

Reply:
Fixed